client/tasks/options/template.js

```diff
@@ -13,7 +14,7 @@ module.exports = function(grunt) {
 
         var config = {
             raven: {dsn: process.env.SUPERDESK_RAVEN_DSN || ''},
-            server: {url: server, ws: ws},
+            server: {url: server, ws: ws, disableEditorToolbar: disableEditorToolbar},
```

Comment:
It would be good to move this option to a separate group instead of mixing with server.